src/main/resources/README.MD

```diff
@@ -0,0 +1,37 @@
+Example Resources
+-----------------
+
+These example resources consist of a number of xml files that can be used
+to become more familiar with the formats used in the EASY system.
+The examples are organised with the same directory structure
+as de XSD files at https://easy.dans.knaw.nl/schemas/
```

Comment:
There is also a `META-INF` directory and a `version file`. Should they (both) be public, if so, what to tell about them?